cfme/tests/services/test_provision_stack.py

```diff
@@ -71,7 +70,7 @@ def dialog(appliance, provider, template):
         choose_type="Text Box",
         default_text_box=service_name
     )
-    dialog = DialogCollection(appliance)
+    dialog = appliance.collections.service_dialogs
     sd = dialog.instantiate(label=dialog_name)
     tab = sd.tabs.instantiate(tab_label="Basic Information")
```

Comment:
Waiting to submit review until I see some update for this tab instantiation.

Reply:
sorry, it took some time because I cannot make Selenium fill catalog right in local FF